src/thunderbots/software/ai/world/field.cpp

```diff
@@ -79,6 +79,18 @@ double Field::defenseAreaLength() const
     return defense_length_;
 }
 
+Rect Field::friendlyDefenseArea() const
+{
+    return Rect(Point(-field_length_ * 0.5, defense_width_ / 2.0),
+                Point(-field_length_ * 0.5 + defense_length_, -defense_width_ / 2.0));
+}
+
+Rect Field::enemyDefenseArea() const
+{
+    return Rect(Point(field_length_ * 0.5, defense_width_ / 2.0),
+                Point(field_length_ * 0.5 - defense_length_, -defense_width_ / 2.0));
```

Comment:
Similar comment to above regarding verbose naming, could we say `Rectangle` instead of `Rect`? I understand that this isn't quite as obfuscated as `Seg`, but it's still not totally clear (ex. do we mean `Rectify`?)

Reply:
I've added this to #40 as well.

Reply:
wontfix, see #40